wordpress-mobile/WordPress-Android
# Issue/7996 actionable empty states (Part 6)

### Fix
This is a code cleanup pull request which refactors drawable resources, adds parameter annotations, removes view casts, and updates deprecated methods for files touched by the Actionable Empty States project.  There is no functionality directly tied to these changes.  This was not specifically included in https://github.com/wordpress-mobile/WordPress-Android/issues/7996, but I felt it was appropriate to address technical debt I came across.

### Test
Since nearly everything is refactoring, any issues are evident while building rather than at runtime.  The `Snackbar.setCallback()` is the only functionality that needs to be tested.
1. Go to ***My Site*** tab.
2. Tap ***Create*** button in bottom navigation.
3. Enter title and content for post.
4. Tap back arrow.
5. Notice new post is in list.
6. Tap ***Trash*** button on newly created post.
7. Notice new post is not in list.
8. Tap ***Undo*** button on snackbar.
9. Notice new post is in list.
10. Swipe down to refresh list.
11. Notice new post is in list.
12. Tap ***Trash*** button on newly created post.
13. Notice new post is not in list.
14. Wait until snackbar is dismissed automatically.
15. Swipe down to refresh list.
16. Notice new post is not in list.

### Review
Only one develop is required to review these changes, but anyone can perform the review.
